src/ext/auto_flush.h

```diff
@@ -0,0 +1,10 @@
+#ifndef DDTRACE_AUTO_FLUSH_H
+#define DDTRACE_AUTO_FLUSH_H
+
+#include <stdbool.h>
+
+#include "compatibility.h"
```

Comment:
Is this inlclude left-over from a refactoring? Nothing in the header uses it.

Reply:
Nice catch!